lib/beaker/dsl.rb

```diff
@@ -79,5 +80,10 @@ module DSL
     include Beaker::DSL::Helpers
     include Beaker::DSL::InstallUtils
     include Beaker::DSL::Patterns
+    include Beaker::DSL::PE
+
+    def self.register(helper_module)
```

Comment:
I think that fixing the current approach—or at least not encouraging directly opening and referencing the beaker class hierarchy—is definitely something we should be doing. However, this `register` method basically just seems to be sidestepping ruby's privatization of `include`, not necessarily stopping people from mucking with the internals. 

Perhaps I'm just not getting it; @kevpl is this in use anywhere yet in any of the beaker libraries, so I could see it in use and get a better grasp of this?